### PR TITLE
Enhanced readability of the README.md file

### PR DESCRIPTION
**Pull request by geekygirl8**
The grammar correctness was absent at some points thus made the required changes to ensure correctness.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x} My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example only English changes, or only Chinese changes, etc.

Closes #XXXXX

### CODE REVIEW

**Review by RandellDawson — DISMISSED** [review]
Edit: I was incorrect regarding the 1st and 2nd changes.

@geekygirl8  Can you please revert the first two changes?  Thank you.

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 [freeCodeCamp.org](https://www.freecodecamp.org) is a friendly community where you can learn to code for free. It is run by a [donor-supported 501(c)(3) nonprofit](https://donate.freecodecamp.org) to help millions of busy adults transition into tech. Our community has already helped more than 10,000 people get their first developer job.
 
-Our full-stack web development and machine learning curriculum is completely free and self-paced. We have thousands of interactive coding challenges to help you expand your skills.
+Our full-stack web development and machine learning curriculum are completely free and self-paced. We have thousands of interactive coding challenges to help you expand your skills.
```
Comment (nhcarrigan) [inline README.md:12]: This, to me, suggests that the web development and machine learning curriculum are separate. There *is* a bit of overlap between the two, and my understanding was the curriculum is unified.

File: README.md [hunk 2]
```diff
@@ -137,7 +137,7 @@ Once you have earned the Responsive Web Design, Algorithms and Data Structures,
 
 #### Legacy Certifications
 
-We also have 4 legacy certifications dating back to our 2015 curriculum, which are still available. All of the required projects for these legacy certifications will remain available on freeCodeCamp.org.
+We also have 4 legacy certifications dating back to our 2015 curriculum, which is still available. All of the required projects for these legacy certifications will remain available on freeCodeCamp.org.
```
Comment (nhcarrigan) [inline README.md:140]: This also changes the reading of the sentence. Before it inferred that the legacy certifications are still available, now it suggests that the *2015 curriculum* is still available.

Comment (RandellDawson) [reply README.md:140]: Yes, I agree with @nhcarrigan.  Upon second reading, it should be "are" since it is related to "4 legacy certifications".

**Comment by ojeytonwilliams** [comment]
Hi, thanks for creating this PR, but this not a change we will be making at this time.

While we cannot accept this contribution, if you would like to help out, take a look at our [contributing guide](https://contribute.freecodecamp.org/) for other things you can do.

**Pull request closed without merging**